//
//  command_center_node.h
//  
//
//  Created by Austin Burch on 11/1/16.
//
//

#ifndef ____command_center_node__
#define ____command_center_node__

#include <iostream>
#include <math.h>
#include "ros/ros.h"
#include "bbot_pkg/link1_msg.h"
#include "bbot_pkg/link2_msg.h"
#include "bbot_pkg/command_center_msg.h"

#define PERIOD_INIT 1000000
#define SPS 100
#define BUFF_SIZE 100
//#include "/home/ubuntu/BBB_lib/BBB_lib.h"

#endif /* defined(____command_center_node__) */



